Makefile

```diff
@@ -1,31 +1,15 @@
-GODEP=$(GOPATH)/bin/godep
+#!/bin/bash
```

Comment:
@tsg It is also in the Makefile from libbeat. Will remove it there too which will then remove it with the next update from all repos.